ui/raidboss/data/06-ew/raid/p8s.ts

```diff
@@ -477,13 +535,177 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: NetRegexes.startsUsing({ id: '794C', source: 'Hephaistos', capture: false }),
       response: Responses.getOut(),
     },
+    {
+      id: 'P8S First Snake Debuff Collect',
+      // BBC = First in Line
+      // BBD = Second in Line,
+      // D17 = Eye of the Gorgon
+      // D18 = Crown of the Gorgon
+      // CFE = Blood of the Gorgon
+      // CFF = Breath of the Gorgon
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: ['BB[CD]', 'D17', 'CFE'] }),
+      condition: (data) => !data.firstSnakeCalled,
+      run: (data, matches) => {
+        const id = matches.effectId;
+        if (id === 'BBC')
+          data.firstSnakeOrder[matches.target] = 1;
+        else if (id === 'BBD')
+          data.firstSnakeOrder[matches.target] = 2;
+        else if (id === 'D17')
+          data.firstSnakeDebuff[matches.target] = 'gaze';
+        else if (id === 'CFE')
+          data.firstSnakeDebuff[matches.target] = 'poison';
+      },
+    },
+    {
+      id: 'P8S First Snake Debuff Initial Call',
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: ['BB[CD]', 'D17', 'CFE'], capture: false }),
+      condition: (data) => !data.firstSnakeCalled,
+      delaySeconds: 0.3,
+      suppressSeconds: 1,
+      response: (data, _matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          firstGaze: {
+            en: 'First Gaze (w/ ${player})',
+          },
+          secondGaze: {
+            en: 'Second Gaze (w/ ${player})',
+          },
+          firstPoison: {
+            en: 'First Poison (w/ ${player})',
```

Comment:
Still reviewing, but `s/Poison/Blood/`? I also call it poison in my group, but the mechanic is called Blood of the Gorgon. I don't know if it makes more sense to call it by the mechanic name?

Reply:
Yeah, we also call it poison.  I'm inclined to leave it as that, unless you feel strongly.

Reply:
I do not